CDP4Common/Helpers/NestedElementTreeGenerator.cs

```diff
@@ -578,5 +578,75 @@ private NestedParameter CreateNestedParameter(ParameterSubscription subscription
 
             return nestedParameter;
         }
+
+        /// <summary>
+        /// Get <see cref="NestedElement.ShortName"/> for an <see cref="ElementDefinition"/> 
+        /// </summary>
+        /// <param name="elementDefinition">The <see cref="ElementDefinition"/></param>
+        /// <param name="option">The <see cref="Option"/></param>
+        /// <param name="domain">The <see cref="DomainOfExpertise"/></param>
+        /// <returns>The <see cref="NestedElement.ShortName"/> if found, otherwise null</returns>
+        public string GetNestedElementShortName(ElementDefinition elementDefinition, Option option, DomainOfExpertise domain)

```

Comment:
the name of the method does not convey what the code is doing, please think of a different name; it also seems that DomainOfExpertise is not being used and can be removed